onnxruntime/core/providers/nuphar/scripts/model_editor.py

```diff
@@ -828,10 +1058,18 @@ def parse_arguments():
     elif args.mode == 'remove_initializers_from_inputs':
         print('Remove all initializers from input for model with IR version >= 4...')
         remove_initializers_from_inputs(args.input, args.output)
+    elif args.mode == 'loop_to_scan':
+        print('Convert Loop to Scan')
+        convert_loop_to_scan_model(args.input, args.output)
+        validate_with_ort(args.input, args.output)
+    elif args.mode == 'save_scan_sub_graph':
+        print('save_scan_sub_graph')
+        save_scan_sub_graph(args.input, args.output)
     else:
         raise NotImplementedError('Unknown mode')
     print('Running symbolic shape inference on output model')
     mp = onnx.load(args.output)
     mp = SymbolicShapeInference.infer_shapes(mp, auto_merge=True)
-    onnx.save(mp, args.output)
+
+    onnx.save(mp, args.output + ".shape_inferenced.onnx")
```

Comment:
>onnx.save(mp, args.output + ".shape_inferenced.onnx") [](start = 4, length = 53)

we should use the output name directly, without postfix.

Reply:
Still need to save the file?

---
In reply to: [539673459](https://github.com/microsoft/onnxruntime/pull/6070#discussion_r539673459) [](ancestors = 539673459)